commands/config.go

```diff
@@ -273,6 +289,61 @@ func configSetCmdF(c client.Client, _ *cobra.Command, args []string) error {
 	return nil
 }
 
+func configEditCmdF(c client.Client, _ *cobra.Command, _ []string) error {
+	config, response := c.GetConfig()
+	if response.Error != nil {
+		return response.Error
+	}
+
+	configB, err := json.MarshalIndent(config, "", "  ")
```

Comment:
nit: What about naming `configB` as `configBytes`? It's a little clearer to me

Reply:
Done!!